osu.Game/Graphics/UserInterface/ScreenTitle.cs

```diff
@@ -12,26 +13,33 @@ namespace osu.Game.Graphics.UserInterface
 {
     public abstract class ScreenTitle : CompositeDrawable, IHasAccentColour
     {
-        private readonly SpriteIcon iconSprite;
+        public const float ICON_WIDTH = ICON_SIZE + icon_spacing;
+
+        protected const float ICON_SIZE = 25;
+
+        private SpriteIcon iconSprite;
         private readonly OsuSpriteText titleText, pageText;
-        public const float ICON_WIDTH = icon_size + icon_spacing;
-        private const float icon_size = 25, icon_spacing = 10;
+
+        private const float icon_spacing = 10;
 
         protected IconUsage Icon
```

Comment:
no more getter? reason?

Reply:
wasn't used

Reply:
/shrug I'm not about to remove many properties in the game just because they're unused, but ok.